Examples/DataRepresentation/Image/ImageAdaptor1.cxx

```diff
@@ -81,87 +83,88 @@ class CastPixelAccessor
 //
 //-------------------------
 
-int main( int argc, char *argv[] )
+int
+main( int argc, char * argv[] )
 {
-  if( argc < 2 )
-    {
+  if ( argc < 2 )
+  {
     std::cerr << "Usage: " << std::endl;
     std::cerr << "ImageAdaptor1   inputFileName" << std::endl;
     return EXIT_FAILURE;
-    }
+  }
 
 
-//  Software Guide : BeginLatex
-//
-//  The CastPixelAccessor class simply applies a
-//  \code{static\_cast} to the pixel values. We now use this pixel accessor
-//  to define the image adaptor type and create an instance using
-//  the standard \code{New()} method.
-//
-//  Software Guide : EndLatex
+  //  Software Guide : BeginLatex
```

Comment:
Does this indentation interfere with the software guide?

Reply:
I don't think so, but we could test.